### DISCUSSION TITLE
error create ionic app

### DISCUSSION THREAD

**Discussion by Aandresmc** [nrwl/nx#9720] [Show and Tell]
tutorial step https://eliteionic.com/tutorials/refactoring-an-ionic-application-into-nx-libraries/

commands was executed:

1. npx create-nx-workspace@latest 
2. settings name workspace -> nx-ionic,  select --preset=apps
3. npm install --save-dev @nxtend/ionic-angular
4. cd nx
5. /nx-ionic -> npx nx generate @nxtend/ionic-angular:application nx-ionic-mobile   

why error?

<img width="1372" alt="image" src="https://user-images.githubusercontent.com/28877837/162085043-1be3dfb6-310c-41b6-8a11-581c49d32883.png">

**Comment by marckassay** [comment]
You might be running into nxtend issue #[595](https://github.com/nxtend-team/nxtend/issues/595).

I just [forked](https://github.com/marckassay/nxtend) the repo ([eduardoRoth/nxtend](https://github.com/eduardoRoth/nxtend)) that has an open PR for that issue, for that I have a build with updated main from the main of 'nxtend-team/nxtend'. You may be familiar with installing an npm package by Github URL, however that doesn't seem to work as-is or at all. 

So if you think this is the reason for your issue, you can clone my repo, build via: `yarn nx run-many --target=build --all`. And do what is mentioned [here](https://nxtend.dev/docs/nxtend/contributing) using the `link` command. Or just copy all of the `dist/packages` folders into your `nx-ionic/node_module` directory.

HTH.
